infrastructure/charts/streamlit/templates/service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "streamlit.fullname" . }}
+  labels:
+    {{- include "streamlit.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - port: {{ .Values.service.port }}
+      targetPort: http
+      protocol: TCP
+      name: http
+  selector:
+    {{- include "streamlit.selectorLabels" . | nindent 4 }}
```

Comment:
We do not need a Service for this chart. Please remove it.

Reply:
Removed [16c58f7](https://github.com/calgo-lab/green-db/pull/91/commits/16c58f74267c8c88732b4aa55a673cf4d37c6fc0)